### DISCUSSION TITLE
Add support for changing repository description on Mobile

### DISCUSSION THREAD

**Discussion by zfauser** [community/community#55246] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.112.1 (99233639)
Device: iPhone 11 (iOS 16.4.1)
</details>

**Comment by aconsuegra** [comment, marked as answer]
Thanks for your feedback! I added this feature request to our backlog, so hopefully we can prioritize in the near future.
